src/test/java/com/google/swarm/tokenization/orc/ORCColumnNameDoFnTest.java

```diff
@@ -37,7 +37,7 @@ public class ORCColumnNameDoFnTest {
 
   @ClassRule public static TemporaryFolder tmpFolder = new TemporaryFolder();
 
-  protected static final String PROJECT_ID = "mock-project-id";
+  protected static final String project_id = "mock-project-id";
```

Comment:
Can this be `private static final String PROJECT_ID` ? Same question for ORCReaderDoFnTest file as well.